webapp/src/main/webapp/js/webconferencing.js

```diff
@@ -0,0 +1,2073 @@
+										currentRoomTitle = roomTitle;
+									} else {
+										currentSpaceId = null;
+										currentRoomTitle = roomTitle;
+									}
+									
+									var $wrapper = $("<div class='callButtonContainerMiniWrapper pull-left' style='display: inline-block;'></div>");
+									var initializer = addCallButton($wrapper, miniChatContext);
+									initializer.done(function($container) {
+										var $button = $container.find(".callButton").first();
+										$button.children(".callButtonTitle, .callTitle, .uiIconMiniArrowDown").remove();
+										$button.removeClass("btn").addClass("uiActionWithLabel btn-mini miniChatCall");
+										$button.children(".uiIconLightGray").removeClass("uiIconLightGray").addClass("uiIconWhite");
+										$container.find(".dropdown-menu").addClass("pull-right");
+										//
+										$titleBar.prepend($wrapper);
+										log("<< initMiniChat DONE " + roomTitle + " for " + currentUser.id);
+									});
+									initializer.fail(function(error) {
+										log("<< initMiniChat ERROR " + roomTitle + " for " + currentUser.id + ": " + error);
+										$miniChat.removeData("minichatcallinitialized");
+									});
+								}).fail(function(err) {
+									log("<< initMiniChat ERROR getting room info from chatServer: ", err);
+								});
+							}, 1000);
+						} else {
+							log("<< initMiniChat CANCELED mini-chat not found or empty");
+						}
+					}
+				};
+				addMiniChatCallButton();
+				// run DOM listener to know when mini chat will be construct (by notif.js script)
+				var MutationObserver = window.MutationObserver || window.WebKitMutationObserver || window.MozMutationObserver;
+				var observer = new MutationObserver(function(mutations) {
+					addMiniChatCallButton();
+				});
+				observer.observe($fullName.get(0), {
+					subtree : false,
+					childList : true,
+					attributes : false,
+					characterData : false
+				});
+			}
+		};
+
+		var userContext = function(userId) {
+			var context = {
+				currentUser : currentUser,
+				userId : userId,
+				isGroup : false,
+				isIOS : isIOS,
+				isAndroid : isAndroid,
+				isWindowsMobile : isWindowsMobile,
+				details : function() {
+					var user = getUserInfoReq(userId);
+					user.fail(function(e, status) {
+						if (typeof status == "number" && status == 404) {
+							log(">> userContext < ERROR get_user " + (e.message ? e.message + " " : "Not found ") + userId + " for " + currentUser.id + ": " + JSON.stringify(e));
+						} else {
+							log(">> userContext < ERROR get_user : " + JSON.stringify(e));
+						}
+					});
+					return user;
+				}
+			};
+			return context;
+		};
+		
+		/**
+		 * Add call button to user's on-mouse popups and panels.
+		 */
+		var initUserPopups = function(compId) {
+			var $tiptip = $("#tiptip_content");
+			// wait for UIUserProfilePopup script load
+			if ($tiptip.length == 0 || $tiptip.hasClass("DisabledEvent")) {
+				setTimeout($.proxy(initUserPopups, this), 250, compId);
```

Comment:
We should list all the js events required by webconferencing, in order to implement them in PLF and avoid relying on timeouts.